packages/ember-routing/lib/system/route.js

```diff
@@ -1855,9 +1855,9 @@ let Route = EmberObject.extend(ActionHandler, Evented, {
     // ...
 
     Router.map(function() {
-        this.route('post', { path: '/post/:post_id' }, function() {
-          this.route('comments', { resetNamespace: true });
-        });
+      this.route('post', { path: '/post/:post_id' }, function() {
+        this.route('comments', { resetNamespace: true });
```

Comment:
Hmm, that's a good point. That isn't needed to demonstrate `modelFor` and would probably be useful to make the example as simple as possible.

Now that I'm looking at the example, setting `post` as a prop on the route also seems strange (`this.set('post', this.modelFor('post'))`). Let me see if I can make the example a little bit more straightforward.